storage/readservice/store.go

```diff
@@ -153,3 +153,12 @@ func (s *store) GetSource(orgID, bucketID uint64) proto.Message {
 		OrganizationID: orgID,
 	}
 }
+
+func (s *store) HasWindowAggregateCapability(ctx context.Context, capability ...*reads.WindowAggregateCapability) bool {
+	return false
+}
+
+// WindowAggregate will invoke a ReadWindowAggregateRequest against the Store.
+func (s *store) WindowAggregate(ctx context.Context, req *datatypes.ReadWindowAggregateRequest) (reads.ResultSet, error) {
+	return nil, errors.New("unimplemented WindowAggregate: https://github.com/influxdata/idpe/issues/6805")
```

Comment:
Include this as a comment. I think it's better if we don't include links in errors even if they should never be hit.